src/app/application-page/application-page.component.html

```diff
@@ -46,6 +46,19 @@
     <th scope="row">Pateikta</th>
     <td>{{application.dateTime}}</td>
   </tr>
+  <tr>
+    <th scope="row">Statusas</th>
+    <td>{{application.status}}</td>
+  </tr>
   </tbody>
+  <div *ngIf="application.status === 'INPROGRESS'">
+    <button name="status" onclick="changeStatus('ACCEPTED')" class="btn btn-success">
```

Comment:
Use Angular event listeners for events, especially angular ones. `(click)` in this case, instead of `onlick`